amaranth/hdl/_ir.py

```diff
@@ -385,6 +391,16 @@ def __init__(self, port, *, i=None, o=None, oe=None, src_loc_at=0, src_loc=None)
         self.src_loc = src_loc or tracer.get_src_loc(src_loc_at)
 
 
+class RequirePosedge(Fragment):
```

Comment:
This should clearly be marked as a temporary measure that can be removed at any point.